content/v2.0/write-data/_index.md

```diff
@@ -76,232 +75,100 @@ InfluxDB accepts the following precisions:
-
-#### Configure Vector
-
-1. On the Vector.dev docs site, see the [InfluxDB Metrics Sink](https://vector.dev/docs/reference/sinks/influxdb_metrics/) page.
-2. Under **Configuration**, click **v2** to view configuration settings.
-3. Scroll down to [How It Works](https://vector.dev/docs/reference/sinks/influxdb_metrics/#how-it-works) for more detail.
-
-#### Configure Apache NiFi
-
-See the _[InfluxDB Processors for Apache NiFi Readme](https://github.com/influxdata/nifi-influxdb-bundle#influxdb-processors-for-apache-nifi)_ for details.
-
-#### Configure OpenHAB
-
-See the _[InfluxDB Persistence Readme](https://github.com/openhab/openhab-addons/tree/master/bundles/org.openhab.persistence.influxdb)_ for details.
-
-#### Configure Apache JMeter
-
-<!-- after doc updates are made, we can simplify to: See the _[Apache JMeter User's Manual - JMeter configuration](https://jmeter.apache.org/usermanual/realtime-results.html#jmeter-configuration)_ for details. -->
-
-To configure Apache JMeter, complete the following steps in InfluxDB and JMeter.
-
-##### In InfluxDB
-
-1. [Find the name of your organization](https://v2.docs.influxdata.com/v2.0/organizations/view-orgs/) (needed to create a bucket and token).
-2. [Create a bucket using the influx CLI](https://v2.docs.influxdata.com/v2.0/organizations/buckets/create-bucket/#create-a-bucket-using-the-influx-cli) and name it `jmeter`.
-3. [Create a token](https://v2.docs.influxdata.com/v2.0/security/tokens/create-token/).
-
-##### In JMeter
-
-1. Create a [Backend Listener](https://jmeter.apache.org/usermanual/component_reference.html#Backend_Listener) using the _InfluxDBBackendListenerClient_ implementation.
-2. In the **Backend Listener implementation** field, enter _org.apache.jmeter.visualizers.backend.influxdb.influxdbBackendListenerClient_
-3. Under **Parameters**, specify the following:
-   - **influxdbMetricsSender**: _org.apache.jmeter.visualizers.backend.influxdb.HttpMetricsSender_
-   - **influxdbUrl**: _http://localhost:9999/api/v2/write?org=my-org&bucket=jmeter_ (include the bucket and org you created in InfluxDB)
-   - **application**: _InfluxDB2_
-   - **influxdbToken**: _my-token_ (include the token you created in InfluxDB)
-   - Include additional parameters as needed.
-4. Click **Add** to add the _InfluxDBBackendListenerClient_ implementation.
-
-#### Configure FluentD 
-
-See the _[influxdb-plugin-fluent Readme](https://github.com/influxdata/influxdb-plugin-fluent)_ for details.
+---
 
 ## Other ways to write data
 
-{{< children >}}
-
-### InfluxDB client libraries
+There are multiple options for writing data into InfluxDB, including both no-code and developer solutions.
 
-Use language-specific client libraries to integrate with the InfluxDB v2 API.
-See [Client libraries reference](/v2.0/reference/api/client-libraries/) for more information.
+  - [No-code solutions](/v2.0/write-data/no-code)
+  - [Development tools](/v2.0/write-data/development-tools)
```

Comment:
You could just use the `{{< children >}}` shortcode here.

Reply:
Won't that also pull in best practices though?

Reply:
Oh, you're right. You can ignore.